editor.planx.uk/src/@planx/components/Review/Review.stories.tsx

```diff
@@ -1,276 +1,54 @@
-    ],
-  },
-  "0q9S6JfcZh": {
-    data: {
-      color: "#EFEFEF",
-      title: "A file upload",
-    },
-    type: 140,
-  },
-  "2zQbbodM7n": {
-    data: {
-      val: "PLANNING_PERMISSION_REQUIRED",
-      text: "Permission needed",
-    },
-    type: 200,
-  },
-  "5tyPXLrKCf": {
-    data: {
-      title: "A text input",
-      description: "Description\n",
-      placeholder: "placeholder\n",
-    },
-    type: 110,
-  },
-  "5uGRLrUdF4": {
-    type: 6,
-  },
-  "6RtX4nCoFx": {
-    data: {
-      tasks: [
-        {
-          title: "A task",
-          description: "www\n",
-        },
-        {
-          title: "Another task",
-          description: "wwww\n",
-        },
-      ],
-    },
-    type: 7,
-  },
-  ANNmSPkOMB: {
-    data: {
-      text: "An option",
-    },
-    type: 200,
-    edges: ["crVbi9HiXP"],
-  },
-  LRIlR6XQpa: {
-    data: {
-      text: "A checklist",
-      allRequired: false,
-    },
-    type: 105,
-    edges: ["aaghpbdKBR", "u2OxJ6DhWK"],
-  },
-  LVsfxkd1uz: {
-    data: {
-      fn: "flag",
-    },
-    type: 500,
-    edges: [
-      "patK4Zx4qg",
-      "gOmbOsj0hw",
-      "2zQbbodM7n",
-      "RyySCuR0z8",
-      "OPeN1FWYfZ",
-      "vJKSgChbLF",
-      "bq63VNlyEB",
-      "wViIvfuIJC",
-    ],
-  },
-  MEfebdfah0: {
-    type: 5,
-  },
-  OPeN1FWYfZ: {
-    data: {
-      val: "PP-NOTICE",
-      text: "Notice",
-    },
-    type: 200,
-  },
-  RyySCuR0z8: {
-    data: {
-      val: "PRIOR_APPROVAL",
-      text: "Prior approval",
-    },
-    type: 200,
-  },
-  TQNqwzSpAS: {
-    data: {
-      title: "A page",
-    },
-    type: 350,
-    edges: ["LRIlR6XQpa", "5tyPXLrKCf"],
-  },
-  Y05nCrMuJV: {
-    data: {
-      text: "A question",
-    },
-    type: 100,
-    edges: ["ANNmSPkOMB", "vEsTPHEQTZ"],
-  },
-  aaghpbdKBR: {
-    data: {
-      text: "A check option",
-    },
-    type: 200,
-  },
-  bq63VNlyEB: {
-    data: {
-      val: "PP-NOT_DEVELOPMENT",
-      text: "Not development",
-    },
-    type: 200,
-  },
-  crVbi9HiXP: {
-    data: {
-      text: "aaa",
-    },
-    type: 300,
-  },
-  gOmbOsj0hw: {
-    data: {
-      val: "IMMUNE",
-      text: "Immune",
-    },
-    type: 200,
-  },
-  lBhoS6eLky: {
-    type: 3,
-  },
-  patK4Zx4qg: {
-    data: {
-      val: "MISSING_INFO",
-      text: "Missing information",
-    },
-    type: 200,
-  },
-  toSzQ8Afxr: {
-    data: {
-      color: "#EFEFEF",
-      title: "A notice",
-      resetButton: false,
-    },
-    type: 8,
-  },
-  u2OxJ6DhWK: {
-    data: {
-      text: "Another check option",
-    },
-    type: 200,
-  },
-  vEsTPHEQTZ: {
-    data: {
-      text: "Another option",
-    },
-    type: 200,
-    edges: ["crVbi9HiXP"],
-  },
-  vJKSgChbLF: {
-    data: {
-      val: "NO_APP_REQUIRED",
-      text: "Permitted development",
-    },
-    type: 200,
-  },
-  wViIvfuIJC: {
-    data: {
-      text: "(No Result)",
-    },
-    type: 200,
-  },
+// TODO improve mock data to include one component of each type
```

Comment:
We should also add  this step to the "adding a component" doc I guess?

Reply:
Maybe, but it's really just one component _if_ shown on the Review page - not sure it's a meaningful step yet at that phase (still interested in thinking through less manual/duplicated mock data etc?) but will keep this one on the radar :ok_hand: